src/core/Microsoft.PowerToys.Settings.UI/Controls/HotkeySettingsControl.xaml.cs

```diff
@@ -168,7 +192,7 @@ private void HotkeyTextBox_GettingFocus(object sender, RoutedEventArgs e)
 
         private void HotkeyTextBox_LosingFocus(object sender, RoutedEventArgs e)
         {
-            if (lastValidSettings != null && (lastValidSettings.IsValid() || lastValidSettings.IsEmpty()))
+            if (lastValidSettings != null && (lastValidSettings.IsValid(_keys) || lastValidSettings.IsEmpty()))

```

Comment:
As I said on the comment above, I will try to remove that private field.

Reply:
I tried not creating the private property and had a problem with infinite loops, just push a PR that uses the public field instead of the private one!